deploy/chart/templates/NOTES.txt

```diff
@@ -0,0 +1,13 @@
+You can provision an EFS-backed persistent volume with a persistent volume claim like below:
```

Comment:
What's this?

Reply:
Actually, that's a leftover from a different internal chart that slipped by. I'll change it.